pkg/client/logs.go

```diff
@@ -146,7 +157,6 @@ func (s *SonobuoyClient) LogReader(cfg *LogConfig) (*Reader, error) {
 	go func(wg *sync.WaitGroup, agg chan *message, errc chan error) {
 		wg.Wait()
 		close(agg)
-		errc <- io.EOF
```

Comment:
No need to do this if we add the 'assume eof' logic above. Otherwise I think we have to worry about another odd edge case: what if every container threw an error, filling the buffered channel, then we tried to add an eof here?

I _think_ that would work OK since the _this_ call may block but eventually the `Read()` call would pull a value off, unblocking this, and closing the error channel. Regardless of if the error channel closed, the program would end rather than hang. Or so I think w/o checking.

It seems the more simple thing is to just close the channels and let the logic happen at the read site where assuming EOF would make sense once both the message and error channels have been closed.